packages/kbn-search-api-panels/utils.ts

```diff
@@ -26,7 +26,27 @@ export const getLanguageDefinitionCodeSnippet = (
   }
 };
 
-export const getConsoleRequest = (code: keyof LanguageDefinition): string | undefined =>
-  code in consoleDefinition && typeof consoleDefinition[code] === 'string'
-    ? (consoleDefinition[code] as string)
-    : undefined;
+export const getConsoleRequest = (
+  code: keyof LanguageDefinition,
+  args?: LanguageDefinitionSnippetArguments
+): string | undefined => {
+  if (code in consoleDefinition) {
+    const codeType:
+      | string
```

Comment:
I think we should either infer the type here or add a type in the `types.ts` file. If we add a new key to the LanguageDefinition with a different type than whats defined here it will break this.